tasks/security.yml

```diff
@@ -97,3 +97,19 @@
 - name: Set SELinux to permisive
   selinux: policy=targeted state=permissive
   when: common_selinux_permisive == true
+
+- name: Increase number of audit log files
```

Comment:
This does not necessarily increase the number, so it would be better to use "Set number...".